css/_components/alert/_alert.css

```diff
@@ -0,0 +1,77 @@
+.alert {
+  align-items: center;
+  background-color: var(--light);
+  border: 1px solid var(--dark);
+  color: var(--dark);
+  display: flex;
+  font-size: var(--fs-xs-1);
+  margin: var(--space-xxxs);
+  padding: var(--pd-sm);
+  width: min(400px, 90%);
+}
+
+.alert__icon--right {
+  cursor: pointer;
+  margin-left: auto;
+}
+
+/* W/ICON */
+.al-wicon .alert__text {
```

Comment:
I suggest .al-wicon to change to .alert-wicon {} as the parent is .alert {}. Can be followed throughout the file